### ISSUE TITLE
Make scrollbar sexy

### ISSUE THREAD

**Issue by isbodand** [isbodand/nhstatistics-core#15]
Make a Perl script that changes the cutedoc's generated HTML have custom WebKit && ms scrollbars because currently, it has the default one which looks horrendous. 
I'll assign this to myself, though anyone who somehow stumbles upon this repo and is a frontend web-dev please experiment with the current index.html and list the changes needed for the custom scrollbar so I can make it into a Perl script to make it be readded automagically

**Comment by isbodand** [comment]
Added an issue on cutedoc's repo for this, though doubt anything will happen anytime soon.

**Comment by isbodand** [comment]
I failed to realize firefox doesn't run on WebKit and it has no official way of changing the scrollbar. I got a workaround and in everything except Chrome, it has an empty scrollbar body on the side without the moving part. I've concluded this is the best I could do, and will ship it as the next release's documentation. Otherwise using some jQuery magic and someone's github project(will have links in the actual docs) everything looks the same. Also closed the aforementioned issue on the cutedoc repo as it is not their fault.
